stock_picking_invoice_link/models/sale_order.py

```diff
@@ -1,6 +1,7 @@
 # Copyright 2013-15 Agile Business Group sagl (<http://www.agilebg.com>)
 # Copyright 2017 Jacques-Etienne Baudoux <je@bcim.be>
 # Copyright 2021 Tecnativa - João Marques
+# Copyright 2022 Antony Herrera - LooErp
```

Comment:
You should only modify copyrights if your contribution is significant (typically not migrations).

Add yourself to CONTRIBUTORS

Reply:
Could you resolve this?